excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/SeaarchCriteriaParserTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.ccd.definition.store.excel.parser;
+
+import com.google.common.collect.Sets;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.SheetName;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchCriteriaEntity;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+class SeaarchCriteriaParserTest extends ParserTestBase {
+
+    private static final String CASE_TYPE_ID_1 = "TestCaseTypeID_1";
+    private static final String CASE_TYPE_ID_2 = "TestCaseTypeID_2";
+
+    public static final String OTHER_CASE_REFERENCE = "OtherCaseReference";
```

Comment:
private ?

Reply:
done in [59c375d1](https://github.com/hmcts/ccd-definition-store-api/pull/1061/commits/59c375d172d81b375f2fe43039955b022afd906e)